activity.go

```diff
@@ -42,26 +42,12 @@ type (
 		ActivityID        string
 		ActivityType      ActivityType
 	}
-)
 
-// RegisterActivity - register a activity function with the framework.
-// A activity takes a context and input and returns a (result, error) or just error.
-// Examples:
-//	func sampleActivity(ctx context.Context, input []byte) (result []byte, err error)
```

Comment:
Can we preserve these comments in the workflow client? they can look at ExecuteActivity, but probably better to have in two places.